extension/src/helpers/stellar.ts

```diff
@@ -1,6 +1,6 @@
 import BigNumber from "bignumber.js";
 import StellarSdk from "stellar-sdk";
-import { isEqual } from "lodash";
+import isEqual from "lodash/isEqual";
```

Comment:
So this is the issue I mentioned to @acharb yesterday: I was unable to `yarn build:extension` locally. It turns out `background`, when not minified, was over 4mb (!). Firefox has a max file size of 4mb, so I had set Webpack to split files if they are over this limit. Unfortunately, we can't do that for `background`. It needs to be one file for a browser to load it in an extension

`lodash` is a huuuuge library. If you import lodash methods this way, it actually just loads the code you need rather than the whole lodash lib. I made this change everywhere I saw a lodash import

During the Manifest upgrade, I'll also be looking at other ways to reduce our bundle size because it's definitely too big right now and we'll continue to run into similar issues in the future